projects/ion/src/lib/chip/chip.component.spec.ts

```diff
@@ -116,6 +116,15 @@ describe('ChipComponent', () => {
       fireEvent.click(screen.getByText(option));
       expect(screen.getAllByText(option)).toHaveLength(1);
     });
+
+    it('should close dropdown when not is multiple and selected an option', async () => {
+      const option = defaultOptions[0].label;
+      const element = screen.getByText('dropdown');
+      fireEvent.click(element);
+      expect(element).toHaveClass('chip-selected');
+      fireEvent.click(screen.getByText(option));
+      expect(element).toHaveClass('chip');
```

Comment:
You can add too a expect to check if any options not is in the document.
`expect(screen.queryAllByText(option)).toHaveLength(0)`